Services/Movies/Movies.API/Controllers/MoviesController.cs

```diff
@@ -40,7 +40,27 @@ public async Task<IActionResult> GetMoviesByPerson(int id)
             var movies = await _repository.GetMoviesByPerson(id);
             return Ok(_mapper.Map<List<Movie>, List<MovieModel>>(movies));
         }
+        
+        [HttpGet("[action]")]
+        public async Task<IActionResult> FilterMovies([FromQuery] FilterMoviesRequest request)
```

Comment:
Na funkcije u kontroleru treba dodati `ProvidesResponseType` anotacije da bi swagger znao sta da ocekuje, npr:
```
[ProducesResponseType(typeof(PaginatedMovieList), StatusCodes.Status200OK)]
```
I povratna vrednost funkcije treba to da reflektuje npr. `Task<ActionResult<PaginatedMovieList>>`.
Ovo cu i ja morati da popravim u mom kodu XD

Reply:
sredicu ovo posle bas me mrzi sad ahahah
pa cu ti da ti zveknem re-request cisto ako naletis na nesto nevezano

Reply:
a ovo ostalo sam ispopravljao